### PR TITLE
Improve unit test coverage

### PR DESCRIPTION
**Pull request by seanf**

### CODE REVIEW

File: zanata-war/src/main/java/org/zanata/webtrans/client/util/JavascriptUtil.java [hunk 1]
```diff
@@ -33,6 +35,7 @@
      * @param varName Variable name.
      * @return The value (as a string) assigned to varName.
      */
+    @CoverageIgnore
```
Comment (carlosmunoz) [inline zanata-war/src/main/java/org/zanata/webtrans/client/util/JavascriptUtil.java]: No reason here?

File: zanata-war/src/main/java/org/zanata/webtrans/client/util/DateUtil.java [hunk 2]
```diff
@@ -53,8 +44,7 @@ public static String formatTime(Date date) {
      */
     public static String formatLongDateTime(Date date) {
         if (date != null) {
-            return DateTimeFormat.getFormat(DATE_TIME_LONG_PATTERN)
-                    .format(date);
+            return dtfLong.format(date);
```
Comment (carlosmunoz) [inline zanata-war/src/main/java/org/zanata/webtrans/client/util/DateUtil.java]: Why did we change it to "not thread safe"?
From what I can see, it was creating its own formatter before which made it thread safe.

File: zanata-war/src/test/java/org/zanata/action/validator/EmailListValidatorTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    @Test
+    public void DIsValid() throws Exception {
+        assertTrue(validator.isValid("d@example", context));
+    }
+
+    @Test
+    public void aBCDIsValid() throws Exception {
+        assertFalse(validator
```
Comment (carlosmunoz) [inline zanata-war/src/test/java/org/zanata/action/validator/EmailListValidatorTest.java]: I thought we were sticking to Assertj for assertions from now on.

File: zanata-war/src/test/java/org/zanata/webtrans/client/util/DateUtilTest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    @BeforeMethod
+    public void before() {
+        aDate = new GregorianCalendar(2013, 11, 25, 1, 2, 3).getTime();
+    }
+
+    @Test
+    public void testFormatShortDate() throws Exception {
+        String s = DateUtil.formatShortDate(aDate);
+        assertEquals(s, "25/12/13 01:02");
```
Comment (carlosmunoz) [inline zanata-war/src/test/java/org/zanata/webtrans/client/util/DateUtilTest.java]: Same comment about assertions here.

File: zanata-war/src/main/java/org/zanata/webtrans/client/util/JavascriptUtil.java (hunk 1 above)
Comment (seanf) [inline zanata-war/src/main/java/org/zanata/webtrans/client/util/JavascriptUtil.java]: That was the first one I put in, before I thought of adding a reason to the annotation.  I'll put it in.

File: zanata-war/src/main/java/org/zanata/webtrans/client/util/DateUtil.java (hunk 2 above)
Comment (seanf) [inline zanata-war/src/main/java/org/zanata/webtrans/client/util/DateUtil.java]: The old version couldn't be tested without using GWTTestCase, which we won't work with TestNG.  GWT code all runs in a single Javascript thread anyway.

File: zanata-war/src/test/java/org/zanata/action/validator/EmailListValidatorTest.java (hunk 3 above)
Comment (seanf) [inline zanata-war/src/test/java/org/zanata/action/validator/EmailListValidatorTest.java]: AssertJ > Hamcrest, but if all you need is a simple boolean check, I think it's overkill.

File: zanata-war/src/test/java/org/zanata/webtrans/client/util/DateUtilTest.java (hunk 4 above)
Comment (seanf) [inline zanata-war/src/test/java/org/zanata/webtrans/client/util/DateUtilTest.java]: Again, it's just equality.  I don't think AssertJ would make this more readable, or produce better failure messages.

**Comment by carlosmunoz** [comment]
:+1: Reviewed

**Pull request merged**